internal/resource/nvml-device.go

```diff
@@ -86,3 +90,37 @@ func (d nvmlDevice) GetTotalMemoryMB() (uint64, error) {
 	}
 	return info.Total / (1024 * 1024), nil
 }
+
+func (d nvmlDevice) GetDisplayMode() (string, error) {
+	info, retVal := d.Device.GetPciInfo()
+	if retVal != nvml.SUCCESS {
+		return "", retVal
+	}
+	var bytes []byte
+	for _, char := range info.BusId {
+		if char == 0 {
+			break
+		}
+		bytes = append(bytes, byte(char))
+	}
+	pciID := strings.ToLower(strings.TrimPrefix(string(bytes), "0000"))
```

Comment:
I recently added a `GetBusID` function to `go-nvml`: https://github.com/NVIDIA/go-nvlib/pull/41 does it make sense to merge that first?

Reply:
Yes having that logic in go-nvlib is better

Reply:
The PR has merged, so we should be able to update the dependecies and use it here.

Reply:
Updated it